tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+                                    ID) to use for transactions.
+* `--monitor-spinup-time-us arg` (=1000000)
+                                    Number of microseconds to wait before 
+                                    monitoring TPS. Defaults to 1000000 
+                                    (1s).
+* `--monitor-max-lag-percent arg` (=5)    Max percentage off from expected 
+                                    transactions sent before being in 
+                                    violation. Defaults to 5.
+* `--monitor-max-lag-duration-us arg` (=1000000)
+                                    Max microseconds that transaction 
+                                    generation can be in violation before 
+                                    quitting. Defaults to 1000000 (1s).
+* `--log-dir arg`                   set the logs directory
+</details>
+
+## Result Reports
+
+### Performance Test
+
+Command used to run test and generate report:
+
+``` bash
+.build/tests/performance_tests/performance_test.py --test-iteration-duration-sec 10 --final-iterations-duration-sec 30 --save-json True
+```
+
+`InitialMaxTpsAchieved` - the max TPS throughput achieved during initial, short duration test scenarios to narrow search window
+`LongRunningMaxTpsAchieved` - the max TPS throughput achieved during final, longer duration test scenarios to zero in on sustainable max TPS
+
+A summary of search scenario conducted and respective results are included.  Each summary includes information on the current state of the overarching search as well as basic results of the individual test that are used to determine whether the basic test was considered successful.
+<details>
+    <summary>Expand Search Scenario Summary Example</summary>
+
+``` json
+    "0": {
+    "success": false,
+    "searchTarget": 25000,
+    "searchFloor": 0,
+    "searchCeiling": 50000,
+    "basicTestResult": {
+        "targetTPS": 25000,
+        "resultAvgTps": 15382.714285714286,
+        "expectedTxns": 250000,
+        "resultTxns": 250000,
+        "tpsExpectMet": false,
+        "trxExpectMet": true,
+        "basicTestSuccess": true,
+        "logsDir": "performance_test/2022-10-12_16-55-27/testRunLogs/performance_test_basic/2022-10-12_16-55-27"
+    }
+    }
+```
+</details>
+
+Finally, the full detail test report for each of the determined max TPS throughput (`InitialMaxTpsAchieved` and `LongRunningMaxTpsAchieved`) runs is included in the full report.  **Note:** In the example full report below, these have been truncated as they are single performance test basic run reports as detailed in the following section [Performance Test Basic Report](#performance-test-basic).  Herein these truncated reports appear like:
+
+<details>
+    <summary>Expand Truncated Report Example</summary>
+
+``` json
+"InitialMaxTpsReport": {
+    "Analysis": {
+<truncated>
+    },
+    "args": {
+<truncated>
+    },
+<truncated>
+}
+```
+</details>
+
```

Comment:
I suggest mentioning that the number of entries in "InitialSearchResults" will vary depending on the TPS range tested.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)